docs/entity_light.md

```diff
@@ -3,14 +3,55 @@ title: Light Entity
 sidebar_label: Light
 ---
 
-> This entry is incomplete. Contribution welcome.
 
-## Properties
+A light entity is a device that controls the brightness, RGB value,color temperature and effects of a light source.
 
-> Properties should always only return information from memory and not do I/O (like network requests). Implement `update()` or `async_update()` to fetch data.
+## Properties
 
 | Name | Type | Default | Description
-| ---- | ---- | ------- | -----------
+| ---- | ---- | ---- | ----
+| brightness | int | None | Return the brightness of this light between 0..255
+| color_temp | int | None | Return the CT color value in mireds.
+| effect | String | None | Return the current effect.
+| effect_list | list | None | Return the list of supported effects.
+| hs_color | list | None | Return the hue and saturation color value [float, float].
+| max_minreds | int | int | Return the warmest color_temp that this light supports.
+| min_mireds | int | int | Return the coldest color_temp that this light supports.
+| supported_features | int | int | Flag supported features.
+| white_value | int | None | Return the white value of this light between 0..255.
+
+
+## Support Feature  
+| Constant | Description 
+|----------|-----------------------
+| 'SUPPORT_BRIGHTNESS' | Controls the brightness of a light source
+| 'SUPPORT_COLOR' | Controls the color a light source shows
+| 'SUPPORT_COLOR_TEMP' | Controls the representation a light source shows based on temperature
+| 'SUPPORT_EFFECT' | Controls the effect a light source shows
+| 'SUPPORT_FLASH' | Controls the duration of a flash a light source shows
+| 'SUPPORT_TRANSITION' | Controls the duration of transitions between color and effects
+| 'SUPPORT_WHITE_VALUE' | Controls the white light a light source shows.
 
 ## Methods
 
+'''python
```

Comment:
You need to use backticks, not quotation marks. 

The preview of this page looks all scrambled https://deploy-preview-228--developers-home-assistant.netlify.com/docs/en/next/entity_light.html

you would also have been able to check this if you clicked preview in GitHub. https://github.com/home-assistant/developers.home-assistant/blob/7b09c0b1e9039c312aff8cc7b5a3332fb3fe440c/docs/entity_light.md